pkg/ansible/flags/flag.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flags
+
+import (
+	"strings"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/ansible/controller"
+	"github.com/spf13/pflag"
+)
+
+// AnsibleOperatorFlags - Options to be used by an ansible operator
+type AnsibleOperatorFlags struct {
+	ReconcilePeriod time.Duration
+	WatchesFile     string
+}
+
+// ApplyToControllerOptions - Apply the flags to the controller options.
+func (f *AnsibleOperatorFlags) ApplyToControllerOptions(options *controller.Options) {
```

Comment:
Why put this logic here? What would you think of instead just putting this in `operator.go`; it has to know about controller stuff anyway, so I think it makes sense for it to take the flags as input (it's already doing that) and use that to set controller options. That way, this whole flags package wouldn't need to know anything about controller options. It just collects user input and makes it available to the operator to interpret.